openwisp_monitoring/check/tasks.py

```diff
@@ -15,17 +17,27 @@ def get_check_model():
 
 
 @shared_task
-def run_checks():
+def run_checks(checks=CHECKS_LIST):
```

Comment:
Why don't we set this to `None` here?

Reply:
Initially I did this, but what if we run management command `run_checks`, In that case it will only execute **ping** and **config applied** check instead of executing all checks in the development.

Reply:
We can modify the management command to pass all three checks in the arguments. 

Running all three checks is not a standard case here. Running `ping` and `config` checks is the standard case. 

If you want to provide configurability to users, you can additionally allow passing the check dotted path as arguments to the run_checks command as follows: 

```
./manage.py run_checks openwisp_monitoring.check.classes.Iperf,openwisp_monitoring.check.classes.ConfigApplied
```

But, this is not our goal now.